extensions/BMO/template/en/default/pages/etiquette.html.tmpl

```diff
@@ -18,133 +18,149 @@
-  relating to [% terms.bugs %]
-  in the [% terms.bug %] itself. Do not send them by private email; no-one else can read
-  them if you do that, and they'll probably just get ignored. If a file
-  is too big for [% terms.Bugzilla %], add a comment giving the file size and contents
-  and ask what to do.
+    <span class="heading">No private email</span>.
+    Do not send comments on [% term.bugs %] by private email to users; 
+    no one else can read them if you do that, and they'll be missed and/or ignored.
+    If an attachment is too big for [% terms.Bugzilla %], add a comment 
+    giving the file size and contents and ask what to do.
   </li>
 </ol>
 
-<h3>2. Changing Fields</h3>
+<h3>Changing Fields</h3>
 
 <ol>
   <li>
-  <span class="heading">No messing with other people's [% terms.bugs %]</span>.
-  Unless you are the [% terms.bug %] assignee, or have some say over the use of their
-  time, never change the Priority or Target Milestone fields. If in doubt,
-  do not change the fields of [% terms.bugs %] you do not own - add a comment
-  instead, suggesting the change.
+    <span class="heading">No messing with other people's [% terms.bugs %]</span>.
+    Unless you are the [% terms.bug %] assignee, or have some say over the use of their
+    time, never change the Priority or Target Milestone fields. If in doubt,
+    do not change the fields of [% terms.bugs %] you do not own—add a comment
+    instead, suggesting the change.
   </li>
 
   <li>
-  <span class="heading">No whining about decisions</span>.
-  If a respected project contributor has marked a [% terms.bug %] as INVALID, then it is
-  invalid. Someone filing another duplicate of it does not change this. Unless
-  you have further important evidence, do not post a comment arguing that an
-  INVALID or WONTFIX [% terms.bug %] should be reopened.
-  </li>
+    <span class="heading">No whining about decisions</span>.
+    If another project contributor has marked a [% terms.bug %] as INVALID, then it is
+    invalid. Filing another duplicate of it does not change this. Unless you have 
+    further evidence to support reopening a [% terms.bug %], do not post a comment 
+    arguing that a [% terms.bug %] resolved as INVALID or WONTFIX should be reopened. 
+  </li> 
 
 </ol>
 
-<h3>3. Applicability</h3>
+<h2>Responding to Violations</h2>
 
-<ol>
-  <li>
-  Some of these rules may not apply to you. If they do not, you will know
-  exactly which ones do not, and why they do not apply. If you are not
-  sure, then they definitely all apply to you.
-  </li>
-</ol>
+<p>
+  If you find a [% terms.Bugzilla %] user violating the [% terms.Bugzilla %] Etiquette 
+  or the <a href="https://www.mozilla.org/about/governance/policies/participation/">Mozilla Community Participation Guidelines</a> 
+  in comments on [% terms.bugs %], please <a href="https://wiki.mozilla.org/BMO/comment_tagging">tag the comments</a> 
+  in question so that an administrator can review them. 
```

Comment:
Here, we are replacing as the initial step an informal process (sending the person a private email) with a semi-formal one (summoning an admin). I think that's a backwards step - if your neighbour is playing loud music, calling the police is not the _first_ thing you do, and is actually less likely to lead to a peaceful resolution.

I think we should take the approach of assuming that people are misinformed rather than malicious in the initial instance, and encourage our community to educate one another in informal ways before we get them to take an escalation path. I know if I were a new contributor, and I said something I thought was enthusiastic and someone called an admin to tell me off, I would feel that as exclusionary and aggressive.